### PR TITLE
Implement Config class

### PR DESCRIPTION
**Pull request by tshemsedinov**
Closes: https://github.com/metarhia/impress/issues/885

### CODE REVIEW

**Comment by tshemsedinov** [comment]
@belochub Please help reimplement JSTP integration to Impress. Now we get `Cannot connect to Cloud Controller: RemoteError: Application not found` after start.

**Comment by tshemsedinov** [comment]
@belochub I have detected the problem, it isn't connected with jstp, so debugging now

File: lib/jstp.js [hunk 1]
```diff
@@ -5,26 +5,28 @@ impress.jstp = jstp;
 
 // Create JSTP Server
 //   config <Object> Impress server worker configuration
-jstp.createServer = config => {
-  const transportModule = config.transport === 'tcp' ? 'net' : config.transport;
-  const transport = api.jstp[transportModule];
-  if (!transport) return null;
+jstp.createServer = serverConfig => {
+  const { name, transport } = serverConfig;
```
Comment (belochub) [inline lib/jstp.js]: The issue is here, `serverConfig` doesn't have the `name` field now:
```js
{ protocol: 'jstp',
  transport: 'tcp',
  address: '127.0.0.1',
  ports: [ 250 ],
  slowTime: 1000,
  bundle: false,
  nagle: false,
  timeout: 30000,
  keepAlive: 5000,
  heartbeatInterval: 0,
  key: undefined,
  cert: undefined,
  applications: undefined,
  port: 250 }
```

Comment (tshemsedinov) [reply lib/jstp.js]: @belochub , thanks, it really helped

**Comment by tshemsedinov** [comment]
Landed in 1f20afd, 88a80e4 and bc58d4a

**Pull request closed without merging**